docs/gpa.md

```diff
@@ -165,6 +170,10 @@ This requires the [Second Stage](https://github.com/open-contracting-extensions/
 * Add *the criteria that will be used to select [the suppliers]* to `tender.selectionCriteria.description` or, if possible, split this into `SelectionCriterion` objects in the `tender/selectionCriteria/criteria` array.
 
 This requires the [Selection Criteria](https://github.com/open-contracting-extensions/ocds_selectionCriteria_extension) extension.
+
+* If *the criteria that will be used to select them* are published as documents:
```

Comment:
Above we used the construction `to select [the suppliers]` to make the pronoun easier to dereference.

Reply:
I don't understand

> to make the pronoun easier to dereference

Reply:
To determine what `them` from the original text refers to :) (in this case, in refers to `[the suppliers]`)

Reply:
Done